static/js/message_fetch.js

```diff
@@ -62,6 +62,10 @@ function process_result(data, opts) {
     if (opts.cont !== undefined) {
         opts.cont(data);
     }
+
+    if (opts.post_scroll_cont !== undefined) {
+        opts.post_scroll_cont(opts);
+    }
```

Comment:
I don't see why we need `post_scroll_cont`; can't we just have `cont` do two things?